content/fast/adaptive-serving-based-on-network-quality/index.md

```diff
@@ -0,0 +1,103 @@
+---
+page_type: guide
+title: Adaptive serving based on network quality
+author: mihajlija
+description: |
+  Using Network Information API to adapt which assets you're serving to users based on the quality of their connection.
```

Comment:
Slight rewrite of this sentence: "Use Network Information API to adapt the assets served to users based on the quality of their connection."